### ISSUE TITLE
XSD Update

### ISSUE THREAD

**Issue by dotsad** [OfficeDev/PnP-Provisioning-Schema#84]
Using master branch:

Even without adding any property elements to existing ProvisioningSchema-2015-12.xsd and running:

`xsd -c ProvisioningSchema-2015-12.xsd /n:OfficeDevPnP.Core.Framework.Provisioning.Providers.Xml.V201512`

We get this error on build.

**Error	33	'OfficeDevPnP.Core.Framework.Provisioning.Providers.Xml.V201512.CustomAction' does not contain a definition for 'RightsSpecified'	D:\GitHub\PnP-Sites-Core\Core\OfficeDevPnP.Core\Framework\Provisioning\Providers\Xml\XMLPnPSchemaV201512Formatter.cs	649	30	OfficeDevPnP.Core**

Basically where is **RightsSpecified** property is coming from if it missing in original xsd file?

**Comment by PaoloPia** [comment]
The [PropertyName]Specified properties are generated by the XSD.EXE tool to handle optional attributes. I will close this issue because it is not an issue, but it is a standard behavior of the serialization engine. We will update the Rights/RightsSpecified handling in the upcoming monthly release. Thanks.
